uglyui/src/services/Server.js

```diff
@@ -48,11 +48,11 @@ export default {
   postsGetOne(id) {
     return get(`/posts/${id}`);
   },
-  postsCreate(post) {
-    return post("/posts", post);
+  postsCreate(pst) {
+    return post("/posts", pst);
```

Comment:
and there's the problem with post creation. I didn't add any unit tests to the client because I thought we'd replace it. but it looks like we aren't and I should.